manifests/local-dind/kind-cluster-v1.12.8.sh

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env bash
```

Comment:
how to move new scripts to `hack/` directory? we can store manifest files in the `manifest` directory.

Reply:
OK
still put the old scripts in manifest directory？

Reply:
> still put the old scripts in manifest directory？
yes, when the kind script is ready to use in local development, we can deprecate current DIND script and related files.

Reply:
Yes, if this is only for e2e test, I think `hack` directory is better.